qa/L0_java_simple_example/pom.xml

```diff
@@ -0,0 +1,38 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.bytedeco.tritonserver</groupId>
+    <artifactId>simple</artifactId>
+    <version>1.5.7-SNAPSHOT</version>
+    <properties>
+        <exec.mainClass>Simple</exec.mainClass>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.bytedeco</groupId>
+            <artifactId>tritonserver-platform</artifactId>
+            <version>2.14-1.5.7-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.8</version>
+        </dependency>
+    </dependencies>
+    <repositories>
+        <repository>
+            <id>sonatype-nexus-snapshots</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>sonatype-nexus-snapshots</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Would it be better to make the changes in javacpp-presets instead? So that the users can use the pom.xml there too

Reply:
I can do that! Would you be okay waiting until tomorrow's meeting to discuss that with Jack? I'm assuming the pom.xml works for Jack and/or he knows why it's not working for us. Based on that conversation, we can update the test to use the javacpp-presets pom.xml (and our instructions for how users can run this example).

Reply:
Since we found a way to build using the javacpp-presets pom, I removed it here.